.devcontainer/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.12.0-alpine3.17
+ENV PYTHONUNBUFFERED 1
+
+# In Visual Studio Code:
+#   edit ./res/config.json and set your API keys
+#   open a terminal and run:
+#       python3 -m http.server
```

Comment:
maybe we could use https://code.visualstudio.com/remote/advancedcontainers/start-processes

Reply:
Yes, sounds good!